OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -80,6 +80,9 @@ public class CargoInfo : TraitInfo, Requires<IOccupySpaceInfo>
 			"A dictionary of [actor id]: [condition].")]
 		public readonly Dictionary<string, string> PassengerConditions = new Dictionary<string, string>();
 
+		[Desc("Change the passengers owner if transport owner changed")]
+		public readonly bool OwnerChangedAffectsPassengers = true;
```

Comment:
Can we please rename this to `ChangePassengerOwner`?

Reply:
But this will cause misunderstand, which may mean change owner when get into transport

Reply:
Ok, then we will need to come up with something different. The current name isn’t suitable because it is against our naming conventions to call trait fields after internal interfaces.

Reply:
InheritOwnerChange?

Reply:
change to InheritOwnerChange